src/modules/translation/languages/en.json

```diff
@@ -1020,6 +1020,7 @@
       "discussion": "Discussion",
       "keep_reviewing": "Keep reviewing",
       "back_to_curation": "Back to curation",
+      "waiting_for_upload": "Some items in the collection are still being uploaded. Please check back soon to proceed with the curation.",
```

Comment:
What do you think of mention where the item needs to be uploaded?

`[...] still being uploaded to the catalysts`

Reply:
I wasn't sure the user knew what that meant (there's no mention of catalysts on the entire site). Maybe is a good idea anyways? I'm not sure if members of the committee know what a Catalyst is (besides...shibu)

Reply:
I guess that will help them to know about the catalyst and also will be easy to report

Reply:
agree, ok! added